src/toil_scripts/gatk_germline/vqsr.py

```diff
@@ -0,0 +1,300 @@
+
+    outputs={'toil.vqsr.vcf': None}
+    docker_call(work_dir = work_dir,
+                env={'_JAVA_OPTIONS':'-Djava.io.tmpdir=/data/ -Xmx{}'.format(config['xmx'])},
+                parameters = command,
+                tool = 'quay.io/ucsc_cgl/gatk:3.5--dba6dae49156168a909c43330350c6161dc7ecc2',
+                inputs=inputs.keys(),
+                outputs=outputs,
+                mock=config['mock_mode'])
+    return job.fileStore.writeGlobalFile(os.path.join(work_dir, 'toil.vqsr.vcf'))
+
+
+def gatk_variant_recalibrator_indel(job, vcf_id, config):
+    """
+    Variant quality score recalibration for INDEL variants.
+
+    :param job: Job instance
+    :param vcf_id str: vcf file store ID
+    :param config dict: pipeline configuration options and shared files
+    :return tuple: recalibration table, tranches file, plots file
+    """
+    job.fileStore.logToMaster('Running GATK VariantRecalibrator (INDEL Mode)')
+    cores = multiprocessing.cpu_count()
+    work_dir = job.fileStore.getLocalTempDir()
+    references = ['genome.fa', 'genome.fa.fai', 'genome.dict', 'mills.vcf']
+    inputs = {key: config[key] for key in references}
+    inputs['toil.vcf'] = vcf_id
+    get_files_from_filestore(job, work_dir, inputs)
+
+    # DP is a recommended annotation, but does not work well with exome data
+    command = ['-T', 'VariantRecalibrator',
+               '-R', 'genome.fa',
+               '-input', 'toil.vcf',
+               '-nt', str(cores),
+               '-resource:mills,known=true,training=true,truth=true,prior=12.0', 'mills.vcf',
+               '-an', 'QD',
+               '-an', 'FS',
+               '-an', 'SOR',
+               '-an', 'ReadPosRankSum',
+               '-an', 'MQRankSum',
+               '-mode', 'INDEL',
+               '-recalFile', 'HAPINDEL.recal',
+               '-tranchesFile', 'HAPINDEL.tranches',
+               '-rscriptFile', 'HAPINDEL.plots',
+               '--maxGaussians', '4',
+               '--minNumBadVariants', '5000']
+
+    if config['unsafe_mode']:
+        command = ['-U', 'ALLOW_SEQ_DICT_INCOMPATIBILITY'] + command
+
+    outputs = {'HAPINDEL.recal': None, 'HAPINDEL.tranches': None, 'HAPINDEL.plots': None}
+    docker_call(work_dir = work_dir,
+                env={'_JAVA_OPTIONS':'-Djava.io.tmpdir=/data/ -Xmx{}'.format(config['xmx'])},
+                parameters = command,
+                tool ='quay.io/ucsc_cgl/gatk:3.5--dba6dae49156168a909c43330350c6161dc7ecc2',
+                inputs=inputs.keys(),
+                outputs=outputs,
+                mock=config['mock_mode'])
+    recal_id = job.fileStore.writeGlobalFile(os.path.join(work_dir, 'HAPINDEL.recal'))
+    tranches_id = job.fileStore.writeGlobalFile(os.path.join(work_dir, 'HAPINDEL.tranches'))
+    plots_id = job.fileStore.writeGlobalFile(os.path.join(work_dir, 'HAPINDEL.plots'))
+    return recal_id, tranches_id, plots_id
+
+
+def gatk_apply_variant_recalibration_indel(job, vcf_id, recal_id, tranches_id, config):
```

Comment:
How much can we factor out between this and `gatk_apply_variant_recalibration_snp`? They're 99% the same.